### PR TITLE
S2U-6 5.1.1.2 Tests & Quizzes: Show number of correct, incorrect and …

### PR DESCRIPTION
**Pull request by JuanDavid102**
…blank answers in export - fix

### CODE REVIEW

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/evaluation/TotalScoresBean.java [hunk 1]
```diff
@@ -265,38 +269,42 @@ public boolean getIsOneSelectionType() {
...
-            // If it has response we will get the answer from publishedAnswerHash and if it is correct or incorrect
-            AnswerIfc answer = (AnswerIfc) publishedAnswerHash.get(item.getPublishedAnswerId());
-            if (!answer.getIsCorrect()) {
-              // For incorrect answers cases
-              resultsAux.set(1, ((int) resultsAux.get(1)) + 1);
-            } else {
-              // For correct answers cases
-              resultsAux.set(0, ((int) resultsAux.get(0)) + 1);
+    if (!this.isEnteredInResults()) {
+      System.out.println(this.isEnteredInResults()? "No, hasta ahora no habia entrado" : "Si, ya hace rato que entro");
```
Comment (jesusmmp) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/evaluation/TotalScoresBean.java:273]: Remove this System.out

Comment (JuanDavid102) [reply samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/evaluation/TotalScoresBean.java]: Sorry for the mistake, now it's removed

**Review by mpellicer — APPROVED** [review]

**Comment by mpellicer** [comment]
@JuanDavid102 Would you mind doing a PR for Master with the same fix, please?

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/evaluation/TotalScoresBean.java [hunk 2]
```diff
@@ -265,38 +269,41 @@ public boolean getIsOneSelectionType() {
...
-            // If it has response we will get the answer from publishedAnswerHash and if it is correct or incorrect
-            AnswerIfc answer = (AnswerIfc) publishedAnswerHash.get(item.getPublishedAnswerId());
-            if (!answer.getIsCorrect()) {
-              // For incorrect answers cases
-              resultsAux.set(1, ((int) resultsAux.get(1)) + 1);
-            } else {
-              // For correct answers cases
-              resultsAux.set(0, ((int) resultsAux.get(0)) + 1);
+    if (!this.isEnteredInResults()) {
+      this.setEnteredInResults(true);
```
Comment (mpellicer) [inline samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/evaluation/TotalScoresBean.java:273]: What does this mean? Like it executed this portion before? Maybe something like this.resultsAlreadyCalculated() ?

**Review by mpellicer — CHANGES_REQUESTED** [review]
Wait for testing results and some minor comments about the IF.

File: samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/evaluation/TotalScoresBean.java (hunk 2 above)
Comment (JuanDavid102) [reply samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/evaluation/TotalScoresBean.java:273]: Yes, it's like you say. But I prefer your variable's name. I will change it

**Comment by jesusmmp** [comment]
Just testing! For me it's ready to merge

**Review by mpellicer — APPROVED** [review]

**Comment by mpellicer** [comment]
Please make a PR for Master, thanks

**Comment by mpellicer** [comment]
@stetsche PLEASE, we need a PR for master, could you cherry-pick this commit and make a single PR for Master? Thanks

**Pull request merged**